### DISCUSSION TITLE
Ignition Temperature Approach for Fire Spread Simulation

### DISCUSSION THREAD

**Discussion by Zaninho23** [firemodels/fds#13575] [Q&A] [answered]
Good morning everyone,
I know that the question I am going to ask is not really about how FDS works, but I wanted to ask your opinion based on your experience. 
I am studying the evolution of a fire inside an industrial building in which wooden pallets are stored in order to determine the temperatures developing at the ceiling.
I have built the model, of which I attach photos from smokeview, in which each pallet has been schematized with an obstruction. 
![image](https://github.com/user-attachments/assets/4b8e12de-3a06-4c1a-bf40-bcdf3d35b8c1)
These obstructions are characterized by the material “wood” and follow, once ignited, a HRRPUA I entered. 
By running the calculation model, it is evident that not all surfaces of the obstructions reach the ignition temperature (250°C) and therefore do not contribute to the total HRR. The result of the model are clear to me, also the heat transfer modes and other issues discussed here: 
https://github.com/firemodels/fds/discussions/12845
What I find incorrect, however, is my approach to the problem. In fact, I believe that in reality if one face of the pallet starts to burn due to pyrolysis and any dripping of materials, it is more conservative to consider that the whole pallet burns and therefore it is wrong to proceed with temperature ignition approach as I have done.
I think it is better to simplify the model a bit and insert obstructions equipped with appropriate VENT and relatice Control device that once they reach the ignition temperature (in this case estimated at 250°C) follow the defined HRRPUA.
What would you guys suggest ?

Thank you very much for any help.

**Comment by drjfloyd** [comment]
I think the larger issue is that you have defined what looks to be on the order of 1 m^3 OBST as being wood. Large dimensional wood is not easy to ignite (one starts a campfire first with small twigs and kindling and not the largest log available). If this is meant to be commodity attached to pallets and then stored on a rack or just stacked, it is likely the carboard and plastic of the commodity that is going to be the drive of firespread. Before jumping straight to a full warehouse domain, I suggest you limit yourself to a small number of columns of commodity and work on better definining properties that represent it.

**Reply by Zaninho23** [reply]
Good evening Dr. Floyd,
thanks for the suggestions. I tried to make a simple model to understand how the heating of obstruction faces changes as two main parameters change. The thickness (mass) and the specific heat. I entered a wall at a fixed temperature of 1000°C radiating 4 obstructions to which I changed the values first of thickness and then of specific heat. In the first analysis I kept the material characteristics fixed and varied the thickness of the surface (0.25; 0.05; 0.01; 0.002m) obtaining these results (150s analysis and monitored with 4 devices the wall temperature):
![image](https://github.com/user-attachments/assets/9c195d0e-ab98-4359-b583-9eea6f075d22)

In the second analysis, I kept the thickness of the surface fixed (0.1m) and varied the specific heat of the material (0.1,1,2.85,10 kJ/(kgK)) obtaining these results (Analysis 150s and monitored with 4 devices the wall temperature):
![image](https://github.com/user-attachments/assets/0f3304e4-0b1c-480b-9f6c-dbf25d0ca13d)

The idea was to go and change the parameters of the Q=m*c*DT function. By varying the thickness and thus the mass, however, I thought the results would be more different. Perhaps there is something wrong. 
I attach the two input files from FDS.
Thank you very much

[SIM5_ver_1.TXT](https://github.com/user-attachments/files/17412756/SIM5_ver_1.TXT)
[SIM5_ver_2.TXT](https://github.com/user-attachments/files/17412758/SIM5_ver_2.TXT)

**Reply by drjfloyd** [reply]
Your target boxes do not have the same view factor of the hot plate.; therefore, you do not have indentical boundary conditions.

**Reply by Zaninho23** [reply]
True ! (Sorry)
So I tried redoing the simulations, leaving only one target cube where I changed the thickness of the surfaces.
In this way, i think, the boundary conditions should be constant and so should the incident heat flux. 
I did 4 simulations with thicknesses of 0.25m 0.05m 0.01m and 0.002m and the results of the device measuring the wall temperature are as follows:

![image](https://github.com/user-attachments/assets/e2746b49-7f46-4e03-9eb7-596dc9431445)

I think there is something else I did wrong, the temperatures that different thickness cubes reach should be more different.
In fact, the mass of the single cell (mesh size 0.25x0.25m) is equal to: 
SIM 1_A: 0.25x0.25x640kg/m3 =10kg
SIM 1_B: 0.25x0.25x0.05x640kg/m3 =2kg
SIM 1_C: 0.25x0.25x0.01x640kg/m3 =0.4kg
SIM 1_D: 0.25x0.25x0.002x640kg/m3 =0.08kg
Is it perhaps related to the backing characteristic of the surface ? (insulated, void, exposed)

I attach the input files of FDS.
Thank you very much
[SIM5_ver_1_D.txt](https://github.com/user-attachments/files/17429691/SIM5_ver_1_D.txt)
[SIM5_ver_1_A.txt](https://github.com/user-attachments/files/17429692/SIM5_ver_1_A.txt)
[SIM5_ver_1_B.txt](https://github.com/user-attachments/files/17429693/SIM5_ver_1_B.txt)
[SIM5_ver_1_C.txt](https://github.com/user-attachments/files/17429695/SIM5_ver_1_C.txt)

**Comment by drjfloyd** [comment]
Why should they  be more different?  Do a hand cacluation for the expected steady state temperatures for the front and back of each surface type. Does it indicate they should be more different?

**Reply by Zaninho23** [reply]
Sorry for all these questions now I think I understand how it works. (I could have done it earlier =( ).
The wall temperature device measures the surface temperature of the obstruction face to which it is attached, and therefore going to change the thickness of the surface will not result in a change on the temperature measured by the device. Obviously, on the other hand, if I change the specific heat of the material, the surface temperature will be affected and will be different (as in fact you can see from the graphs above). If I in addition to the wall temperture I measure the back wall temperature here it will be affected by the thickness of the material.
Now i have a question; Is the ignition_temperature the one measured by the wall temperature sensor i.e. is it the surface temperature of the solid ?
thank you as always

**Reply by drjfloyd** [reply, marked as answer]
It is the WALL TEMPERATURE.

When you have questions that can be answerd with a simple input file, a great way to become a better FDS user is to make a file and test. You could in this case have an input file for a small domain with an OBST that has a SURF with an IGNITION_TEMPERATURE. Then you can look to see when does it ignite and is the temperature at which it ignites the surface temperature, some in depth temperature, or the gas temperature. As you get proficient with FDS it will only take you a few minutes to make and run such files which is generally going to get you an answer much quicker than posting here.
